Assets/Scripts/ViewModels/GameViewModel.cs

```diff
@@ -110,7 +110,8 @@ public enum Difficulty
 
 		// TODO: For now, skip straight to starting the game since i turned off crew selection
 		GUI_startMainGame();
-
+		//play the intro for the game
+		time_Line_controll.play();
```

Comment:
can you maybe comment this part out? i think it's good that we're getting the progress you made on the opening cutscene pushed to git, but we need to make sure it won't play since we never finished it. will have to revisit that later

Reply:
sure